chart/values.yaml

```diff
@@ -0,0 +1,3071 @@
+
+  ## Maps API config
+  ##
+  MAPS_API_V3_RESOURCE_URL_HOST="${ONPREM_DOMAIN}"
+  MAPS_API_V3_RESOURCE_URL_ALLOWED_HOSTS="${ONPREM_DOMAIN}"
+
+  ## Import API config
+  ##
+  IMPORT_TENANT_ID="${ONPREM_TENANT_ID}"
+  IMPORT_WORKER_PROCESSING_DIR="/tmp/import-worker"
+
+  IMPORT_PUBSUB_TENANT_BUS_TOPIC="projects/${ONPREM_GCP_PROJECT_ID}/topics/tenant-bus"
+  IMPORT_PUBSUB_TENANT_BUS_SUBSCRIPTION="projects/${ONPREM_GCP_PROJECT_ID}/subscriptions/tenant-bus-import-sub"
+
+## @param commonSecretConfiguration [object] Sensitive configuration script that will be run in all CARTO deployments
+##
+commonSecretConfiguration: |
+  # CARTO self hosted sensitive information
+  ENCRYPTION_SECRET_KEY="<FILL_ME>"
+
+## @section accounts-www Deployment Parameters
+##
+accountsWww:
+  ## accounts-www image
+  ## @param accountsWww.image.registry accounts-www image registry
+  ## @param accountsWww.image.repository accounts-www image repository
+  ## @param accountsWww.image.tag accounts-www image tag (immutable tags are recommended)
+  ## @param accountsWww.image.pullPolicy accounts-www image pull policy
+  ## @param accountsWww.image.pullSecrets accounts-www image pull secrets
+  ##
+  image:
+    registry: gcr.io
+    repository: carto-onprem-artifacts/accounts-www
+    tag: "2021.12.20"
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+  ## @param accountsWww.replicaCount Number of accounts-www replicas to deploy
+  ##
+  replicaCount: 1
+  ## @param accountsWww.containerPorts.http accounts-www HTTP container port
+  ##
+  containerPorts:
+    http: 80
+  ## Configure extra options for accounts-www containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ## @param accountsWww.livenessProbe.enabled Enable livenessProbe on accounts-www containers
+  ## @param accountsWww.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param accountsWww.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param accountsWww.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param accountsWww.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param accountsWww.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 30
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
```

Comment:
I'd recommend that CARTO team revisit the default seconds we're setting for every probe (readiness, liveness and startup) per component. They'll have more context about the reasonable times to wait for the micro-services to be ready/healthy/started.

Reply:
Ok!